tests/contrib/django/test_django_snapshots.py

```diff
@@ -156,28 +157,42 @@ def test_psycopg_query_default(client, psycopg2_patched):
 
 
 @pytest.mark.skipif(django.VERSION < (3, 0, 0), reason="ASGI not supported in django<3")
-@pytest.mark.snapshot(
+@snapshot(
     variants={
         "30": (3, 0, 0) <= django.VERSION < (3, 1, 0),
         "31": (3, 1, 0) <= django.VERSION < (3, 2, 0),
         "3x": django.VERSION >= (3, 2, 0),
     },
+    token_override="tests.contrib.django.test_django_snapshots.test_asgi_200",
 )
-def test_asgi_200(daphne_client):
-    resp = daphne_client.get("/")
-    assert resp.status_code == 200
-    assert resp.content == b"Hello, test app."
+@pytest.mark.parametrize("django_asgi", ["application", "channels_application"])
+def test_asgi_200(django_asgi):
+    with daphne_client(django_asgi) as client:
+        resp = client.get("/")
+        assert resp.status_code == 200
+        assert resp.content == b"Hello, test app."
 
 
 @pytest.mark.skipif(django.VERSION < (3, 0, 0), reason="ASGI not supported in django<3")
-@pytest.mark.snapshot(
+@snapshot()
+def test_asgi_200_simple_app():
+    # Generates an empty snapshot
```

Comment:
can we get a little more of a description here.

why does this generate an empty snapshot?

Reply:
this also isn't accurate since you added a `TraceMiddleware` around it

Reply:
Yup, I forgot to update that comment